src/Symfony/Component/FeatureFlag/Provider/ChainProvider.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\FeatureFlag\Provider;
+
+/**
+ * @experimental
+ */
+final class ChainProvider implements ProviderInterface
```

Comment:
At this stage Bundle is not fully integrated. But when it does I would make this the default Provider with the InMemory as only provider. Makes it easier to integrate new ones.

Reply:
Removing this class means that people will not be able to see their own provider injected automatically into the `ChainProvider` with this first version.

What will be the benefice to create a dedicated PR later for that?

Reply:
Not what I wanted to say. Sorry for the confusion. I fully agree with what you said ! What I meant is exactly this : to make sure that the checker gets a chainprovider with the at least the inmemory and any other tagged provider. But I saw that it was already implemented so closing this. Thanks